src/components/Navbar.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div id="navbar" class="navbar">
+    <div class="nav">
+      <div class="logo-container">
+        <img class="logo" alt="Mesh logo" src="../assets/mesh-logo.png">
+      </div>
+      <div class="link-container">
+        <span class="date">
+          <img class="calendar" alt="Calendar icon" src="../assets/calendar.svg">
+          {{ date }}
+        </span> |
+        <span class="name">Hi, Nikki</span> |
+        <router-link to="/">Home</router-link> |
+        <router-link to="/about">About</router-link>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      date: '05/15/2015'
```

Comment:
We should talk at some point about what we have in mind for the date. Will we use it to attach feedback to a particular date? (If so we'll need to collaborate with the BE folks so they know they need to store that info; I don't think it's in the schema right now.)

Reply:
We don't have to have it at all. I just thought it was a nice touch to have today's date at the top of the page. We can take it out or whatever you want to do. 
With that said, I think our navigation will be turning into a header and the left Modules menu will become our "navigation". That makes the most sense, since we don't really need any links in the top nav bar.